deploy/charts/approver-policy/templates/deployment.yaml

```diff
@@ -76,9 +74,6 @@ spec:
       {{- with .Values.volumes }}
```

Comment:
```suggestion
      {{- with .Values.volumes }}
      volumes:
```

Reply:
@wallrj can you move the `volumes:` key into this with conditional?
Same for the `volumeMounts:` key

Reply:
Done. Checked that volumes fields are now omitted by default and rendered when volumes and volumeMounts are supplied:

```sh
$ helm template _bin/scratch/image/cert-manager-approver-policy-v0.13.0-alpha.0-19-g3d69b70a086b9b.tgz | grep -i volume
```

```sh
$ helm template _bin/scratch/image/cert-manager-approver-policy-v0.13.0-alpha.0-19-g3d69b70a086b9b.tgz --set-json 'volumes=[{"name": "temp-dir", "emptyDir": {}}]' --set-json 'volumeMounts=[{"name": "temp-dir", "mountPath": "/tmp"}]' | grep -C 5 -i volume

          - --webhook-host=0.0.0.0
          - --webhook-port=10250
          - --webhook-service-name=cert-manager-approver-policy
          - --webhook-ca-secret-namespace=default
        volumeMounts:
        - mountPath: /tmp
          name: temp-dir

        resources:
          {}

        securityContext:
          allowPrivilegeEscalation: false
          capabilities: { drop: ["ALL"] }
          readOnlyRootFilesystem: true
      volumes:
      - emptyDir: {}
        name: temp-dir

      hostNetwork: false
      dnsPolicy: ClusterFirst
```